service/javadsl/integration-client/src/main/java/com/lightbend/lagom/javadsl/client/integration/LagomClientFactory.java

```diff
@@ -209,7 +209,7 @@ public void addStopHook(Callable<? extends CompletionStage<?>> hook) {
         ServiceInfo serviceInfo = ServiceInfo.of(serviceName);
 
         // ServiceClientLoader
-        CircuitBreakers circuitBreakers = new CircuitBreakers(actorSystem, new CircuitBreakerConfig(configuration),
+        CircuitBreakers circuitBreakers = new CircuitBreakersImpl(actorSystem, new CircuitBreakerConfig(configuration),
```

Comment:
Oh, that's unfortunate. My assumption was that this was used internally only as it's being initialised for the users in the guice module / macwire components. 

We can't keep the construct because it's now a trait. The risk is for user updating Lagom but not updating the ConductR bundle.